Phoenix/PHApp.m

```diff
@@ -166,7 +167,27 @@ - (BOOL) activate {
 
 - (BOOL) focus {
 
-    return [self.app activateWithOptions:NSApplicationActivateIgnoringOtherApps];
+    if (![NSProcessInfo isOperatingSystemAtLeastBigSur]) {
+
+        return [self.app activateWithOptions:NSApplicationActivateIgnoringOtherApps];
+    } else {
+
+            // Source https://stackoverflow.com/a/65464683/525411
+        if (!self.app || self.app.processIdentifier == -1) {
+            return false;
+        }
+    
+        
+
+        ProcessSerialNumber process;
+        OSStatus error = GetProcessForPID(self.app.processIdentifier, &process);
+        if (error) {
+            return false;
+        }
+
+        error = SetFrontProcessWithOptions(&process, kSetFrontProcessFrontWindowOnly);
+        return (error == 0) ? true : false;
```

Comment:
You can just return the result of the conditional (i.e. no need for the explicit ternary).